scripts/migration/ca-migration/migrate_ca

```diff
@@ -688,6 +710,7 @@ check_cert_in_pod() {
     fi
     if ! ${rootmap[${SN}]}; then
       echo "${POD}.${NS} does not trust ${CA_CERT}"
+      ERR="true"
```

Comment:
I think using 0 and 1, then -eq or -ne to check, is safer.

Reply:
done